Grbl_Esp32/src/Stepper.cpp

```diff
@@ -265,24 +262,17 @@ static void stepper_pulse_func() {
                 st.exec_block_index = st.exec_segment->st_block_index;
                 st.exec_block       = &st_block_buffer[st.exec_block_index];
                 // Initialize Bresenham line and distance counters
-                st.counter_x = st.counter_y = st.counter_z = (st.exec_block->step_event_count >> 1);
+                // XXX the original code only inits X, Y, Z here, instead of n_axis.  Is that correct?
```

Comment:
Weird, I'd say it's a bug judging from what I see while looking at counter_x vs counter_a/b/c.
In fact, `st.counter_a` is never initialized in the whole code base, except for the memset in `st_reset`.

The ' // TODO ABC' right below might be an explanation.